UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -403,11 +403,18 @@ public void SetupRoom(bool calculateGrid = true)
             }
 
             // If tiles exist at all, grab the most used tile size and use that as our grid size
+            // If a default starting grid is set in the settings, use that instead
```

Comment:
This comment is slightly misleading, as in this code block the starting grid is *not* get from the settings as the grid isn't even recalculated in that case.